lms/static/js/certificates/views/certificate_bulk_whitelist.js

```diff
@@ -31,7 +31,7 @@
             return Backbone.View.extend({
                 el: DOM_SELECTORS.bulk_exception,
                 events: {
-                    'change #browseBtn': 'chooseFile',
+                    'change #browseBtn-bulk-exc-students': 'chooseFile',
```

Comment:
I'm confused about this change because I don't see that browseBtn-bulk-exc-students appears elsewhere in this PR (I would have expected the HTML to change accordingly).